accounts/src/main/java/com/jungeeks/dto/ChildDto.java

```diff
@@ -0,0 +1,23 @@
+package com.jungeeks.dto;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.util.List;
+
+
+@Data
+@AllArgsConstructor
```

Comment:
What type of constructor do you need? There are 3 types now

Reply:
fix